index.html

```diff
@@ -16,19 +17,22 @@
     </script>
     <title>Hello, I am Adam Chang</title>
 </head>
-<body>
+<body>    
     <header class="head-container">
          
         <a href=""><h2 id="name">Adam Chang</h2></a>
-        <nav>
+        <nav id="nav">
             <ul class="nav-list">
-                <li class="nav-list-items"><a href="">Portfolio</a></li>
-                <li class="nav-list-items"><a href="">About</a></li>
-                <li class="nav-list-items"><a href="">Contact</a></li>
-                <li class="nav-list-items"><a href=""><img src="Images/mail.svg" alt=""></a></li>
+                <li class="nav-list-items"><a class="nav_a" href="#works-container">Portfolio</a></li>
+                <li class="nav-list-items"><a class="nav_a" href="#about_me">About</a></li>
+                <li class="nav-list-items"><a class="nav_a" href="#contact-form">Contact</a></li>
+                <li class="nav-list-items"><a class="nav_a" href=""><img src="Images/mail.svg" alt=""></a></li>
 
             </ul>            
-            <img  id="menu-btn" src="Images/Union.png" alt="menu button"> 
+            <button id="menu-btn" type="button">
+                <img class="menu-icon"  src="Images/Union.png" alt="menu button">
+                <img class="close-icon" src="Images/close.png" alt="close button">
+            </button> 
```

Comment:
- [ ] Kindly make sure your hamburger menu button remains after a user clicks the mobile menu
![Screenshot from 2021-10-04 19-20-51](https://user-images.githubusercontent.com/65860678/135888054-38cf465e-0934-47e8-af1a-bd167eb3554c.png)
.

Reply:
I have made the changes, thank you for the review!

Reply:
This issue is well resolved! ✔️

Reply:
Thank you, Robert!